libs/product/src/models/composite-product-item.ts

```diff
@@ -19,20 +19,50 @@ export enum DaffCompositeProductItemInputEnum {
  * composite product, or neither if the item is optional.
  */
 export interface DaffCompositeProductItem {
+	/**
```

Comment:
The docs uses `the item`, `this item`, and `this product` interchangeably. I think it'd be best to pick one. (I prefer `the item`)

Reply:
I think I made these changes in favor of "the item". Let me know of any I missed.